conf/defaults.ini

```diff
@@ -256,7 +256,7 @@ cookie_secure = false
 cookie_samesite = lax
 
 # set to true if you want to allow browsers to render Grafana in a <frame>, <iframe>, <embed> or <object>. default is false.
-allow_embedding = false
+allow_embedding = true
```

Comment:
We don't make changes to conf/defaults.ini - that's a repository-owned file.  Instead, we make changes to a `custom.ini` but this is also part of our CICD deployment process.

@matthewzhaocc - are the changes we make available somewhere for litebox to easily see or make use of?

Reply:
Oh ok, i'll revert the default.ini changes!